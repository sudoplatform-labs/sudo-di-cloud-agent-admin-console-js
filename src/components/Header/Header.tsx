import React from 'react';
import styled from 'styled-components';
import { theme } from '../../theme';

const Container = styled.div`
  background-color: #fff;
  color: ${theme.greys.fjord};
  display: flex;
  height: 64px;
  justify-content: space-between;
  padding: 0 24px;
`;

const DocsLinkContainer = styled.div`
  display: flex;
  align-items: center;
  text-align: center;
  height: 32px;
  margin-right: 16px;
  padding-right: 16px;
  a {
    font-size: 14px;
    color: ${theme.colors.sudoBlue};
  }
`;

export const AppHeader: React.FC = () => (
  <Container className="header">
    <div></div>
    <DocsLinkContainer>
      <a
        href="https://docs.sudoplatform.com/"
        target="_blank"
        rel="noopener noreferrer">
        Developer Docs
      </a>
    </DocsLinkContainer>
  </Container>
);
