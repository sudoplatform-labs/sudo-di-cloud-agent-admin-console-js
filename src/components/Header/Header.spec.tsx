import React from 'react';
import { shallow } from 'enzyme';

import { AppHeader as Header } from './Header';

describe('Header', () => {
  it('should render correctly', () => {
    const component = shallow(<Header />);

    expect(component).toMatchSnapshot();
  });
});
