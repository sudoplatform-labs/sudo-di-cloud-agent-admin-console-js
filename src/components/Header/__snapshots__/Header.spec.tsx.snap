// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<styled.div
  className="header"
>
  <div />
  <styled.div>
    <a
      href="https://docs.sudoplatform.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Developer Docs
    </a>
  </styled.div>
</styled.div>
`;
